mlflow/gateway/providers/cohere.py

```diff
@@ -336,24 +336,38 @@ def __init__(self, config: RouteConfig) -> None:
         self.cohere_config: CohereConfig = config.model.config
 
     @property
-    def auth_headers(self) -> dict[str, str]:
+    def headers(self) -> dict[str, str]:
```

Comment:
Renaming a property is a breaking change?

Reply:
Nope, this is only used within the class now.